### PR TITLE
added devices for using in automation

### PR DESCRIPTION
**Pull request by michapr**
- changed options from select to pulldown
- added option for wind speed units selection
- added option for refresh selection with limits
- added option for create temp/hum devices
- added option for create wind devices

local_weather_mod_2

### CODE REVIEW

File: modules/OpenWeather/index.js [hunk 1]
```diff
@@ -109,17 +218,30 @@ OpenWeather.prototype.fetchExtendedWeather = function(instance) {
...
 					sunrise = Math.round(res.data.sys.sunrise) * 1000,
 					icon_sunrise = "http://openweathermap.org/img/w/01d.png",
 					sunset = Math.round(res.data.sys.sunset) * 1000,
 					icon_sunset = "http://openweathermap.org/img/w/01n.png";
 				
+				if(self.config.units_wind === "km/h") {
+					windspeed = Math.round(windspeed * 3.6);
+					windgust = Math.round(windgust * 3.6);
+				}	
+				if(self.config.units_wind === "mph")  {
```
Comment (PoltoS) [inline modules/OpenWeather/index.js:240]:
```suggestion
				} else if (self.config.units_wind === "mph")  {
```

File: modules/OpenWeather/index.js [hunk 2]
```diff
@@ -109,17 +218,30 @@ OpenWeather.prototype.fetchExtendedWeather = function(instance) {
...
 					temp = Math.round((self.config.units === "celsius" ? main.temp - 273.15 : main.temp * 1.8 - 459.67) * 10) / 10,
+					hum = main.humidity,
 					icon = "http://openweathermap.org/img/w/" + weather[0].icon + ".png",
 					flag = "http://openweathermap.org/images/flags/" + country.toLowerCase() + ".png",
 					sunrise = Math.round(res.data.sys.sunrise) * 1000,
 					icon_sunrise = "http://openweathermap.org/img/w/01d.png",
 					sunset = Math.round(res.data.sys.sunset) * 1000,
 					icon_sunset = "http://openweathermap.org/img/w/01n.png";
 				
+				if(self.config.units_wind === "km/h") {
```
Comment (PoltoS) [inline modules/OpenWeather/index.js:236]:
```suggestion
				if (self.config.units_wind === "km/h") {
```

File: modules/OpenWeather/index.js [hunk 3]
```diff
@@ -109,17 +218,30 @@ OpenWeather.prototype.fetchExtendedWeather = function(instance) {
...
+					windspeed = wind.speed,
+					windgust = wind.gust,
+					winddeg = wind.deg,
 					clouds = res.data.clouds,
 					country = res.data.sys.country,
 					weatherData = {'main': main,'weather': weather, 'wind': wind, 'clouds': clouds},
 					temp = Math.round((self.config.units === "celsius" ? main.temp - 273.15 : main.temp * 1.8 - 459.67) * 10) / 10,
+					hum = main.humidity,
 					icon = "http://openweathermap.org/img/w/" + weather[0].icon + ".png",
 					flag = "http://openweathermap.org/images/flags/" + country.toLowerCase() + ".png",
```
Comment (PoltoS) [inline modules/OpenWeather/index.js:230]:
```suggestion
					icon = "https://openweathermap.org/img/w/" + weather[0].icon + ".png",
					flag = "https://openweathermap.org/images/flags/" + country.toLowerCase() + ".png",
```

File: modules/OpenWeather/module.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
 	"homepage": "http://razberry.z-wave.me",
 	"icon": "icon.png",
 	"moduleName":"OpenWeather",
-	"version": "1.2.0",
+	"version": "1.2.1",
```
Comment (PoltoS) [inline modules/OpenWeather/module.json:9]:
```suggestion
	"version": "1.3.0",
```

This is not a minor change

File: modules/OpenWeather/module.json [hunk 5]
```diff
@@ -59,17 +81,35 @@
 			},
 			"units": {
 				"label": "__l_units__",
-				"optionLabels": ["°C", "°F"]
+				"type": "select"
+			},
+			"units_wind": {
+				"type": "select"
```
Comment (PoltoS) [inline modules/OpenWeather/module.json:87]: Worth adding optionLabels to allow translations of km/h, mph,... But optional

**Review by PoltoS — COMMENTED** [review]
Thank you for this great contribution! Please accept a few suggestions (if there are no objections). And up to you to add the optionLabels translation for km/h

File: modules/OpenWeather/module.json (hunk 5 above)
Comment (michapr) [reply modules/OpenWeather/module.json:87]: I don't think that's really necessary, as it's understandable...

**Pull request merged**